README.md

```diff
@@ -1,8 +1,33 @@
 # Prebid Cache
 
-This application stores short-term data for use in Prebid.
+This application stores short-term data for use in Prebid. It exists to support Video Ads from Prebid.js, as well as prebid-native
```

Comment:
Suggested rewording to something like:

"This application stores short-term data for use in the Prebid ecosystem, primarily targeting video, native, and AMP formats."

or perhaps

"This application stores short-term data for use in Prebid Server and Prebid.js, primarily targeting video, native, and AMP formats."

Reply:
I like: "This application stores short-term data for use in Prebid Server and Prebid.js, primarily targeting video, native, and AMP formats."